guides/hack/02-source-code-fundamentals/04-program-structure.md

```diff
@@ -32,6 +32,13 @@ have been `run`, `do_it`, or `make_magic`. (What makes `main` the start-up funct
 A file can import another file via [inclusion](script-inclusion.md).
 
 ## Legacy Files
+```yamlmeta
+{
+  "metaonly messages": [
+    "All Hack source code at Meta should use the .php file extension and Hack strict mode (plain <?hh header line)."
+  ]
+}
+```
```

Comment:
Does adding this really belong on this PR?

Reply:
Someone internally reached out to me to write a note and I realized that we should rebrand as well. So I kind of just started adding and it became something else. You're right in that it should be separated.